app/models/deployment.rb

```diff
@@ -82,13 +82,21 @@ def deployment_channel
   end
 
   def deployment_channel_name
```

Comment:
I don't think we should do this here, the model already knows about `["is_internal"]`. See the helper comment above.

Reply:
this function is used in the notifications, we should show the `Internal` identifier in notifications also, no?